RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,240 @@
+## Subject Line
+[Blox] Deploy release - v$$blox_version$$
+
+## Activity Details
+This activity is to: Release Blox version v$$blox_version$$ by pushing our code up to the master branch on GitHub, and pushing new Docker images up to Docker Hub.
```

Comment:
Changed verbiage to `by merging the dev branch into the master branch on GitHub, and pushing new Blox images up to Docker Hub`. It's not really `updating` as these are new images being pushed up to Docker Hub.